pelicun/assessment.py

```diff
@@ -777,23 +766,24 @@ def calculate_asset(
         num_stories: int,
         component_assignment_file: str | None,
         collapse_fragility_demand_type: str | None,
-        add_irreparable_damage_columns: bool,
         component_sample_file: str | None,
-    ):
+        *,
```

Comment:
This is to avoid what they call the "boolean trap". Here is an example:
```
calculate_sum(my_dataframe, True)
```
What's True doing here? We don't know unless we go find the documentation for `calculate_sum`, and look at the second positional argument. But if it was
```
calculate_sum(my_dataframe, across_rows=True)
```
it would be less ambiguous. It's still bad. What would `False` do? Across columns, or sum all elements? And so on. But it's better than before. So we move all boolean arguments to the back and make them keyword only in our function definitions, enforcing all uses of the functions to spell them out.

Reply:
Smart, I like it.